docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -227,3 +230,34 @@ module.exports = {
   },
 };
 ```
+
+==== `fork`and `unlocked_accounts`
+
+Allows to fork a chain and unlock specified accounts on it. Useful for running integration tests with other protocols.
+
+
+```javascript
+// test-environment.config.js
+
+module.exports = {
+  node: { // Options passed directly to Ganache client
+    fork: 'https://mainnet.infura.io/v3/{token}@{blocknumber}, // An url to Ethereum node to use as a source for a fork
+    unlocked_accounts: ['0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B'], // Array of addresses specifying which accounts should be unlocked.
+    allowUnlimitedContractSize: false // Allows unlimited contract sizes.
+  },
+};
+```
+
+==== `allowUnlimitedContractSize`
+
+Allows unlimited contract sizes while debugging. By enabling this flag, the check within the EVM for contract size limit of 24KB (see EIP-170) is bypassed. Enabling this flag will cause ganache-cli to behave differently than production environments.
```

Comment:
```suggestion
Allows unlimited contract sizes. By enabling this flag, the check within the EVM for contract size limit of 24kB (see https://eips.ethereum.org/EIPS/eip-170[EIP-170]) is bypassed. Useful when testing unoptimized contracts that wouldn't be otherwise deployable.

WARNING: Enabling this flag causes Ganache to behave differently from production environments.
```